variables.tf

```diff
@@ -26,6 +26,12 @@ variable "location" {
   default     = ""
 }
 
+variable "storage_class" {
+  description = "The storage class of the GCS bucket. Defaults to 'Standard' if nothing is specified here. See https://cloud.google.com/storage/docs/storage-classes."
```

Comment:
All the available options are not mentioned in the doc. I guess you can consider including this line too: `Available options include "STANDARD", "MULTI_REGIONAL", "REGIONAL", "NEARLINE", "COLDLINE", "ARCHIVE"`

Reply:
Regional Multi-Regional is not a storage "**CLASS**". Ref: https://cloud.google.com/storage/docs/storage-classes

Reply:
@syedrakib If you scroll down on the same doc, you will see a section for [Additional classes](https://cloud.google.com/storage/docs/storage-classes#legacy). But it seems like they are legacy and recommendation is to use STANDARD.